maker/maker.go

```diff
@@ -97,6 +97,7 @@ func FormatFieldList(src []byte, fl *ast.FieldList) []string {
 			names[i] = n.Name
 		}
 		t := string(src[l.Type.Pos()-1 : l.Type.End()-1])
+		t = strings.ReplaceAll(t, pkgName+".", "")
```

Comment:
This LOC is the actual change. The rest is tests and passing around the `pkgName` param.